views/js/tpl/item-progress.tpl

```diff
@@ -0,0 +1,18 @@
+<span class="progress">
+{{#with section}}
+    {{#if position}}<span class="state section-progress">{{__ 'section'}} {{position}}{{#if total}}/{{total}}{{/if}}</span>{{/if}}
+    {{#if label}}<span class="state section-label">{{label}}</span>{{/if}}
+{{/with}}
+{{#with item}}
+    {{#if position}}<span class="state item-progress">{{__ 'item'}} {{position}}{{#if total}}/{{total}}{{/if}}</span>{{/if}}
+    {{#with time}}
+        {{#if display}}
+    <span class="item-time">
+            {{#if elapsedStr}}<span class="elapsed" title="{{__ 'Elapsed time'}}">{{elapsedStr}}</span>{{/if}}
+        <span class="icon icon-time"></span>
+            {{#if remainingStr}}<span class="remaining" title="{{__ 'Remaining time'}}">{{remainingStr}}</span>{{/if}}
+    </span>
```

Comment:
I is tru that the customer asked for  `time remaining, and time spent on current item` but now that I see both time displyed separated with the icon, I find it not very clear. What about only displaying the elapsed time and the total time, something like : 
:hourglass: 1 min / 15 min
Open to discussion...